frontend/packages/db-structure/src/parser/prisma/parser.ts

```diff
@@ -157,4 +157,38 @@ function normalizeConstraintName(constraint: string): ForeignKeyConstraint {
   }
 }
 
+function convertToPostgresColumnType(
+  type: string,
+  enums: readonly DMMF.DatamodelEnum[],
+): string {
+  const enumNames = enums.map((e) => e.name)
+
+  if (enumNames.includes(type)) {
+    return 'enum'
+  }
+
+  switch (type) {
+    case 'String':
+      return 'text'
```

Comment:
If `@db.Char(x)` is defined, it must be referenced.

```hcl
model Model {
  /* ... */
  myField String @db.Char(x)
}
```

But I don't think this needs to be addressed now.

Reply:
Yes.You're absolutely right. We also don't support ``real`` types, ``UUID`` types, and others. 
I'll try developing them in the next PR!

Reply:
@FunamaYukina No need to perfect it now. I'll leave it up to you to decide how far you want to go!